src/DssSpell.sol

```diff
@@ -56,163 +49,181 @@ contract DssSpellAction is DssAction {
+
+    // ---------- Reduce PSM-PAX-A Debt Ceiling & Disable DC-IAM ----------
+    VatAbstract internal immutable vat = VatAbstract(DssExecLib.vat());
+
+    // ---------- RETH-A Offboarding Parameters Finalization ----------
+    address internal immutable MCD_SPOT = DssExecLib.spotter();
+
+    // ---------- SBE parameter changes ----------
+    address internal immutable MCD_VOW  = DssExecLib.vow();
+    address internal immutable MCD_FLAP = DssExecLib.flap();
+
+    // ---------- Approve HV Bank (RWA009-A) DAO Resolution ----------
+    // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/21
+
+    // Approve DAO resolution hash QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w
+    // Note: by the previous convention it should be a comma-separated list of DAO resolutions IPFS hashes
+    string public constant dao_resolutions = "QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w";
+
+    address internal constant RWA009_A_INPUT_CONDUIT_URN_USDC = 0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1;
+
+    // ---------- AVC members compensation Q4 2023 ----------
+    GemAbstract internal immutable MKR          = GemAbstract(DssExecLib.mkr());
+    address internal constant IAMMEEOH          = 0x47f7A5d8D27f259582097E1eE59a07a816982AE9;
+    address internal constant DAI_VINCI         = 0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a;
+    address internal constant OPENSKY_2         = 0xf44f97f4113759E0a57756bE49C0655d490Cf19F;
+    address internal constant ACREDAOS          = 0xBF9226345F601150F64Ea4fEaAE7E40530763cbd;
+    address internal constant FHOMONEYETH       = 0xdbD5651F71ce83d1f0eD275aC456241890a53C74;
+    address internal constant RES               = 0x8c5c8d76372954922400e4654AF7694e158AB784;
+    address internal constant HARMONY_2         = 0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3;
+    address internal constant LIBERTAS          = 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f;
+    address internal constant SEEDLATAMETH      = 0x0087a081a9B430fd8f688c6ac5dD24421BfB060D;
+    address internal constant ROOT              = 0xC74392777443a11Dc26Ce8A3D934370514F38A91;
 
     // ---------- Trigger Spark Proxy Spell ----------
-    // Spark Proxy: https://github.com/marsfoundation/sparklend/blob/d42587ba36523dcff24a4c827dc29ab71cd0808b/script/output/1/primary-sce-latest.json#L2
     address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0x2f2c514137173bc98B3699A0d291f7593637c596;
+    address internal constant SPARK_SPELL = 0xa3836fEF1D314d4c081C2707a7664c3375F29b61;
 
     function actions() public override {
+        // ---------- Stability Fee Changes ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445
+
+        // Increase the ETH-A Stability Fee (SF) by 1.49%, from 5.25% to 6.74%.
+        DssExecLib.setIlkStabilityFee("ETH-A", SIX_PT_SEVEN_FOUR_PCT_RATE, true);
```

Comment:
previous patterns include `/* doDrip = */ ` for the last argument of the method ☝️

Reply:
Thanks, added it! But to be fair, `/* doDrip = */` is present only in 8 spells out of approximately 38 where `setIlkStabilityFee` has been used 🙂